### ISSUE TITLE
CF compliant xarray variables

### ISSUE THREAD

**Issue by jessecusack** [modscripps/velosearaptor#16]
When creating xarray objects it would be good to follow CF: https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html

This mainly just means adding standard names (when defined), e.g.

Code like:
```python
ds.u.attrs = dict(long_name="u", units="m/s")
```

Would become:
```python
ds.u.attrs = dict(standard_name="eastward_sea_water_velocity", long_name="u", units="m s-1")
```
Changes would be needed in `madcp.py` and `io.py`.

**Comment by gunnarvoet** [comment]
Yes, sounds like a great idea! In the long run this should also make it easier to interface with data coming from other packages?!

**Comment by jessecusack** [comment]
Right! And could go straight into long-term archives with minimal additional effort. 

For this to work I think it would need a dictionary that maps variable names to CF standard names.

**Comment by gunnarvoet** [comment]
Closed with #50.
